moped-editor/src/views/projects/projectView/ProjectComponents.js

```diff
@@ -220,64 +197,73 @@ const ProjectComponents = () => {
             {componentsAvailable && (
               <ClickAwayListener onClickAway={handleComponentClickAway}>
                 <List className={classes.root}>
-                  {data.moped_proj_components.map((component, compIndex) => {
-                    const projComponentId = component.project_component_id;
-                    return (
-                      <ListItem
-                        key={"mcListItem-" + projComponentId}
-                        role={undefined}
-                        dense
-                        button
-                        onClick={() => handleComponentClick(projComponentId)}
-                        className={
-                          projComponentId === selectedComp
-                            ? classes.componentItemBlue
-                            : classes.componentItem
-                        }
-                      >
-                        <ListItemIcon>
-                          <Checkbox
-                            edge="start"
-                            className={classes.listItemCheckbox}
-                            checked={projComponentId === selectedComp}
-                            tabIndex={-1}
-                            disableRipple
-                            inputProps={{ "aria-labelledby": null }}
-                          />
-                        </ListItemIcon>
-                        <ListItemText
-                          id={"mcListItemText-" + projComponentId}
-                          primary={component?.moped_components?.component_type}
-                          secondary={
-                            (component?.moped_components?.component_subtype ??
-                              "") +
-                            [
-                              ...new Set(
-                                component.moped_proj_components_subcomponents.map(
-                                  mpcs =>
-                                    mpcs.moped_subcomponent.subcomponent_name
-                                )
-                              ),
-                            ]
-                              .sort()
-                              .join(", ")
+                  {data.moped_proj_components.map(
+                    (projectComponent, compIndex) => {
+                      const projComponentId =
+                        projectComponent.project_component_id;
+                      return (
+                        <ListItem
+                          key={"mcListItem-" + projComponentId}
+                          role={undefined}
```

Comment:
I know this was just carryover from before, but do you know why we set the button's role to be undefined?

Reply:
no idea! 🤔